codepulse/src/main/scala/com/secdec/codepulse/tracer/ProjectFileUploadHandler.scala

```diff
@@ -22,6 +22,8 @@ package com.secdec.codepulse.tracer
 import java.io.File
 import net.liftweb.http.BadResponse
 import net.liftweb.http.LiftResponse
+import net.liftweb.http.RedirectWithState
```

Comment:
nitpick: spuriously added imports?

Reply:
I removed the two unnecessary import statements from ProjectFileUploadHandler.scala.